contracts/evoting/mod_test.go

```diff
@@ -3,7 +3,6 @@ package evoting
 // todo: json marshall and unmarshall branch is are not covered yet
```

Comment:
I understand we don't necessarily have the time, but if Emilien could make a pass to split the tests into smaller units, I really believe it would be a significant improvement and it'd give a better example to the next batch of students. 😇